src/game_renderer/Program.cpp

```diff
@@ -114,3 +114,11 @@ GLint Program::getUniform(const string &name) const {
    }
    return uniform->second;
 }
+
+std::string Program::getName() const {
+   return progName;
+}
+
+void Program::setName(const string &name) {
```

Comment:
Yes it should, I thought we got rid of all of the "using namespace std" lines :/